commands/leaderboard.js

```diff
@@ -0,0 +1,41 @@
+var fs = require("fs");
+
+module.exports = {
+  leaderboard: function (message, roster) {
+
+    if(message.content.split(" ").length !== 2) { // we want users to input how many
```

Comment:
Let's make the default number `10` - add the command spec to commands.json to define how to use the command

Reply:
done.
thank you for doing the needful.